GPU/Directx9/TextureCacheDX9.cpp

```diff
@@ -604,17 +604,20 @@ void TextureCacheDX9::SetTexture(bool force) {
 		lastBoundTexture = INVALID_TEX;
 	}
 
-	u32 texaddr = gstate.getTextureAddress(0);
+	u8 level = 0;
+	if (gstate.getTexLevelMode() == GE_TEXLEVEL_MODE_CONST)
+		level = (gstate.texlevel >> 20) & 0xF;
+	u32 texaddr = gstate.getTextureAddress(level);
```

Comment:
I didn't consider this before, but picking the texaddr here will cause the game to create separate textures for each level. This means that the Tactics Ogre style of usage will actually probably perform well, while the choppy video issue from that other game will not be solved. 

Not sure if I like this or not...